Examples/BookingSystem.AspNetCore/Feeds/FacilitiesFeeds.cs

```diff
@@ -43,80 +45,150 @@ x.Modified < (DateTimeOffset.UtcNow - new TimeSpan(0, 0, 2)).UtcTicks)
 
                 var query = db
                     .SelectMulti<FacilityUseTable, SellerTable>(q)
-                    .Select(result => new RpdeItem<FacilityUse>
+                    .Select(result =>
                     {
-                        Kind = RpdeKind.FacilityUse,
-                        Id = result.Item1.Id,
-                        Modified = result.Item1.Modified,
-                        State = result.Item1.Deleted ? RpdeState.Deleted : RpdeState.Updated,
-                        Data = result.Item1.Deleted ? null : new FacilityUse
+                        var faker = new Faker() { Random = new Randomizer((int)result.Item1.Id) };
+                        var isGoldenRecord = faker.Random.Bool();
```

Comment:
So does this mean that half of records (in Lorem Fitsum mode) will be golden? I thought it was a lower rate than that in the previous LF app? Maybe I misremember

Reply:
Yeh fair, changed to 75%